test/cmd/newapp.sh

```diff
@@ -126,6 +158,10 @@ os::cmd::expect_success 'oc delete template ruby-helloworld-sample'
 os::cmd::expect_success_and_text 'oc new-app mysql --name=db' 'db'
 os::cmd::expect_success 'oc new-app https://github.com/openshift/ruby-hello-world -l app=ruby'
 os::cmd::expect_success 'oc delete all -l app=ruby'
+# check for error when template JSON file has errors
+jsonfile="${TMPDIR:-/tmp}/foo.json"
+echo '{"test"}' > "${jsonfile}"
+os::cmd::expect_failure_and_text "oc new-app '${jsonfile}'" "error: unable to load template file \"${jsonfile}\": invalid character '}' after object key"
```

Comment:
In practice, don't you think users will miss this error message when it is followed by a block of 10+ lines telling how new-app works?
I believe people may read the new-app instructions once, and in all subsequent runs our brains just pattern-matches the block of text and ignore it.